### ISSUE TITLE
Bugged Exits

### ISSUE THREAD

**Issue by Dugmaren** [torilmud/issues#356]
```
A Crossroads in the Grass Fields [70:14636]
Exits: - North [70:14637] - East [-1:-1]  - South [70:14635] - West [-1:-1] 
```
wildland_trails.wld

**Comment by Inames** [comment]
Probably placeholder exits where something was intended to be placed to expand it? I mean it is a crossroads after all.

**Comment by marthammor** [comment]
Yup. Thinking it was from an old project involving homeland that was started a long time ago and bits of it got moved to mainmud at some point.

**Comment by torilmud-silvanus** [comment]
Digging into it:
- These exits are to Delimbiyr Road which is not currently live
- That zone is mostly just a road area it looks like
  - If it was loaded Delimbiyr would also connect to the Tradeway South at 418316
  - Delimbiyr also is designed to connect to Secomber Trail, which currently is just a road area

- Further investigation needed to determine if there is any downside to moving these roads to main
- @gruumsh is probably the best person to know what the background and future intent for these areas are
